From c6a09dc1ef9458649e911c5da140a2702c1834b3 Mon Sep 17 00:00:00 2001
From: sgnanase <sundar.gnanasekaran@intel.com>
Date: Mon, 20 Mar 2017 10:39:12 +0530
Subject: [PATCH 2/3] Summary : Keep make kernel-flinger as default target
 instead of GRUB

Issue   : kernel-flinger & GRUB can't be compiled at same time
          kernel-flinger requires SPARSE_IMG [whereas, GRUB doesnt]
          and vendor.img is missing in flashfiles.zip

Fix     : keep flashfiles as default & GRUB as optional in Buildbot.

Jira    : AIA-235

Change-Id: I1b22c6f894507b04e8e60ac96f6b003d166437a6
Signed-off-by: sgnanase <sundar.gnanasekaran@intel.com>
---
 tasks/publish.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tasks/publish.mk b/tasks/publish.mk
index 782840d..9ffa529 100644
--- a/tasks/publish.mk
+++ b/tasks/publish.mk
@@ -214,5 +214,5 @@ PUBLISH_GOALS := $(DEFAULT_GOAL)
 endif
 
 .PHONY: publish
-publish: publish_mkdir_dest $(PUBLISH_GOALS) publish_ifwi publish_gptimage publish_firmware_symbols $(PUB_OSAGNOSTIC_TAG) $(PUB_CMCC_ZIP) publish_androidia_image
+publish: publish_mkdir_dest $(PUBLISH_GOALS) publish_ifwi publish_gptimage publish_firmware_symbols $(PUB_OSAGNOSTIC_TAG) $(PUB_CMCC_ZIP) publish_flashfiles
 	@$(ACP) $(DIST_DIR)/* $(publish_dest)
-- 
1.9.1

